packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml

```diff
@@ -370,7 +370,12 @@ Resources:
         Fn::If:
           - HasCloudFormationExecutionPolicies
           - Ref: CloudFormationExecutionPolicies
-          - Ref: AWS::NoValue
+          - Fn::If:
+            - HasTrustedAccounts
+            # The CLI will prevent this case from occurring
+            - Ref: AWS::NoValue
+            # The CLI will advertise that we picked this implicitly
+            - - Fn::Sub: "arn:${AWS::Partition}:iam::aws:policy/AdministratorAccess"
```

Comment:
I know. It is correct though:

```
- - Fn::Sub: '....'                                                                           
▲ ▲        ▲                                                                                  
│ │        └─── This makes the list element an object                                         
│ │                                                                                           
│ └─── This starts a new list (because there's no bullet underneath it's a single element)    
│                                                                                             
└───── This is the Else branch of the { Fn::If: [Cond, Then, Else] }                          
```